src/main/java/core/Game.java

```diff
@@ -94,25 +98,6 @@ public final void reset() {
         resetStats();
     }
 
-    /**
-     * Resets the game. Sets up the game state to the initial state as described by game rules,
-     * and initialises all players.
-     *
-     * @param newRandomSeed - random seed is updated in the game parameters object and used throughout the game.
-     */
-    public final void reset(long newRandomSeed) {
```

Comment:
I didn't check the full class, but this is not needed anymore?

Reply:
Nothing used it..so I aggressively deleted it as non-used cruft. The other two reset() methods were all used regularly...I can put it back if you feel it is useful.